### PR TITLE
Update spirit model and create user model

### PR DESCRIPTION
**Pull request by dave7892000**

### CODE REVIEW

File: app/controllers/spirits_controller.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def create
     if @spirit.save
       flash[:notice] = "Spirit was created successfully"
       redirect_to spirits_path
-    else render :new, status: :unprocesable_entity
+    else render :new, status: :unprocessable_entity
```
Comment (kopchickm) [inline app/controllers/spirits_controller.rb]: Let's put the `render` on its own line. This is valid as-is but typically in Ruby we keep the `else` line empty of anything else.

That reminds me...we should set up rubocop to catch things like this.

File: app/views/spirits/new.html.erb [hunk 2]
```diff
@@ -12,6 +12,8 @@
     <%= form.label :abv %>
     <%= form.number_field :abv %>
   </div>
+  <div>
+    <%= form.select :user_id, User.all.map {|u| [u.name, u.id]} %>
```
Comment (kopchickm) [inline app/views/spirits/new.html.erb:16]: Looks like you're missing a closing div tag here.

File: test/controllers/main_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```
Comment (kopchickm) [inline test/controllers/main_controller_test.rb]: I think this test file shouldn't exist. the `test` directory is usually used for minitest, a different testing framework from Rspec.

**Comment by kopchickm** [comment]
This looks really good! I'm guessing your tests are broken, though, now that you have users in the system.

File: app/controllers/spirits_controller.rb (hunk 1 above)
Comment (dave7892000) [reply app/controllers/spirits_controller.rb]: Done! And yes, rubocop soon.

File: app/views/spirits/new.html.erb (hunk 2 above)
Comment (dave7892000) [reply app/views/spirits/new.html.erb:16]: div div div

**Pull request merged**